jupyter_book/book_template/content/contributing.md

```diff
@@ -0,0 +1,227 @@
+# Contributing to Jupyter Book
+
+Welcome to the `jupyter-book` repository! We're excited you're here and want to contribute.
+
+These guidelines are designed to make it as easy as possible to get involved.
+If you have any questions that aren't discussed below, please let us know by opening an [issue][link_issues]!
+
+Before you start you'll need to set up a free [GitHub][link_github] account and sign in.
+Here are some [instructions][link_signupinstructions].
+
+Already know what you're looking for in this guide? Use the TOC to the right
+to navigate this page!
+
+## Joining the conversation
+
+`jupyter-book` is a young project maintained by a growing group of enthusiastic developers&mdash; and we're excited to have you join!
+Most of our discussions will take place on open [issues][link_issues].
+
+As a reminder, we expect all contributors to `jupyter-book` to adhere to the [Jupyter Code of Conduct][link_coc] in these conversations.
+
+## Contributing through GitHub
+
+[git][link_git] is a really useful tool for version control.
+[GitHub][link_github] sits on top of git and supports collaborative and distributed working.
+
+You'll use [Markdown][markdown] to chat in issues and pull requests on GitHub.
+You can think of Markdown as a few little symbols around your text that will allow GitHub
+to render the text with formatting.
+For example you could write words as bold (`**bold**`), or in italics (`*italics*`),
+or as a [link][rick_roll] (`[link](https://https://youtu.be/dQw4w9WgXcQ)`) to another webpage.
+
+GitHub has a helpful page on
+[getting started with writing and formatting Markdown on GitHub][writing_formatting_github].
+
+
+## Understanding issues, milestones and project boards
+
+Every project on GitHub uses [issues][link_issues] slightly differently.
+
+The following outlines how the `jupyter-book` developers think about these tools.
+
+**Issues** are individual pieces of work that need to be completed to move the project forwards.
+A general guideline: if you find yourself tempted to write a great big issue that
+is difficult to describe as one unit of work, please consider splitting it into two or more issues.
+
+Issues are assigned [labels](#issue-labels) which explain how they relate to the overall project's
+goals and immediate next steps.
+
+
+### Issue labels
+
+The current list of labels are [here][link_labels] and include:
+
+* [![Help Wanted](https://img.shields.io/badge/-help%20wanted-159818.svg)][link_helpwanted] *These issues contain a task that a member of the team has determined we need additional help with.*
```

Comment:
For some reason the rendering of these badges is off -- they're floating in the center of the page (and by "for some reason" I mean "I think because of our CSS rules"). I don't love how they look as a result. WDYT of removing them ?